src/objectTranslations/Contact-ja.objectTranslation

```diff
@@ -3481,11 +3485,11 @@
     <label>性別</label>
     <picklistValues>
       <masterLabel>Female</masterLabel>
-      <translation>女性</translation>
+      <translation>Female</translation>
```

Comment:
These values should remain translated. Only the specific custom label should have had its translation reversed.
```suggestion
      <translation>女性</translation>
```